frame/dapps-staking/src/mock.rs

```diff
@@ -173,17 +183,3 @@ impl ExternalityBuilder {
         ext
     }
 }
-
-pub(crate) fn get_dapp_staking_events() -> Vec<pallet_dapps_staking::Event<TestRuntime>> {
```

Comment:
@Maar-io not sure why you removed this, I kind of liked it since it simplifies the testing.